scene/register_scene_types.cpp

```diff
@@ -589,7 +589,7 @@ void register_scene_types() {
 	GDREGISTER_CLASS(CPUParticles3D);
 	GDREGISTER_CLASS(Marker3D);
 	GDREGISTER_CLASS(RootMotionView);
-	GDREGISTER_ABSTRACT_CLASS(SkeletonModifier3D);
+	GDREGISTER_CLASS(SkeletonModifier3D);
```

Comment:
```suggestion
	GDREGISTER_VIRTUAL_CLASS(SkeletonModifier3D);
```
From latest animation meeting.

Reply:
I'm torn about the extension vs base class. reduz seemed to think it was ok to do it as a base class, and I think it will be easier to understand as `GDREGISTER_VIRTUAL_CLASS(SkeletonModifier3D)` so I am going with that.